### PR TITLE
Notification feature

### PR DESCRIPTION
**Pull request by mitalirs**
## Related Issuse

- no notification feature

Closes: #435

#### Describe the changes you've made

1. added notification model
2. added getNotifications route in the new controller file
3. added the corresponding route in the notification route file

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.(Specially for Backend)
- [x] My changes generate no new warnings.

## Screenshots

|        Original         |          Updated           |
| :---------------------: | :------------------------: |
| **no getNotifications route** | <b>Screenshot (1129)</b> |

### CODE REVIEW

File: server/src/routes/notification.routes.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+
+const {
+    requireSignin,
+    userMiddleWare
+} = require("../common-middleware/common-middleware");
+const notificationController = require("../controllers/notification.controller")
+const router = express.Router();
+
+// @route             Get signin/signup
```
Comment (codewithvk) [inline server/src/routes/notification.routes.js:10]: Remove this.

File: server/src/controllers/notification.controller.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+const Notification = require("../models/notification.model")
+const responseHandler = require("../helpers/responseHandler");
+
+const ObjectID = require('mongodb').ObjectID;
+
+const getNotifications = async(req, res) => {
```
Comment (codewithvk) [inline server/src/controllers/notification.controller.js:6]: You can add here description which you mention [here](https://github.com/purnima143/Kurakoo/pull/458/files#diff-cb4a11377e70fceae76797806915f650f162bbb61a18e2cc1883de7ce929d16cR10)

**Review by codewithvk — CHANGES_REQUESTED** [review]

File: server/src/routes/notification.routes.js (hunk 1 above)
Comment (mitalirs) [reply server/src/routes/notification.routes.js]: removed

**Comment by codewithvk** [comment]
@mitalirs Resolve conflict as well.

**Comment by mitalirs** [comment]
done

> @mitalirs Resolve conflict as well.

**Pull request merged**